doc/asciidoc/similarity-cosine.adoc

```diff
@@ -185,10 +186,42 @@ include::scripts/similarity-cosine.cypher[tag=query]
 |===
 | `cuisine`
 | Italian
-| French
+| Lebanese
 |===
 // end::query[]
 
+[[algorithms-similarity-cosine-cypher-projection]]
+== Cypher projection
+
+If the similarity lists are very large they can take up a lot of memory.
+For cases where those lists contain lots of values that should be skipped, you can use the less memory intensive approach of using Cypher statements to project the graph instead.
+
+.Set `graph:'cypher'` in the config:
```

Comment:
the title of this example says `graph:'cypher'`, but the example uses `graph: "cypher"`